src/integrationTest/resources/responses/caseworker-edit-record-listing-about-to-submit-response.json

```diff
@@ -0,0 +1,73 @@
+{
+  "data": {
+    "caseLinks": [],
+    "allCaseworkerCICDocument": [],
+    "allCaseworkerCICDocumentUpload": [],
+    "newCaseworkerCICDocument": [],
+    "newCaseworkerCICDocumentUpload": [],
+    "editCicaCaseDetails": {},
+    "contactParties": {},
+    "caseBundles": [],
+    "cicCaseHearingList": {
+      "value": {
+        "label": "1 - Final - 30 Aug 2024 10:00"
+      },
+      "valueLabel": "1 - Final - 30 Aug 2024 10:00"
+    },
+    "cicCaseHearingNotificationParties": [],
+    "cicCaseRespondentName": "Appeals team",
+    "cicCaseRespondentOrganisation": "CICA",
+    "cicCaseRespondentEmail": "appeals.team@cica.gov.uk",
+    "cicCaseFullName": "Test Name",
+    "cicCaseSchemeCic": "Year2012",
+    "cicCaseFirstDueDate": "",
+    "stayIsCaseStayed": "No",
+    "hearingList": [
+      {
+        "id": "1",
+        "value": {
+          "hearingStatus": "Listed",
+          "hearingType": "Final",
+          "hearingFormat": "Face to Face",
+          "shortNotice": "Yes",
+          "hearingVenues": {
+          },
+          "regionList": {
+          },
+          "venueNotListedOption": [],
+          "roomAtVenue": "G.01",
+          "addlInstr": "Ground floor",
+          "date": "2024-08-14",
+          "hearingTime": "10:00"
+        }
+      },
+      {
+        "id": "2",
+        "value": {
+          "hearingStatus": "Listed",
+          "hearingType": "Interlocutory",
+          "hearingFormat": "Face to Face",
+          "shortNotice": "Yes",
+          "hearingVenues": {
+          },
+          "regionList": {
+          },
+          "venueNotListedOption": [],
+          "roomAtVenue": "G.01",
+          "addlInstr": "Ground floor",
+          "date": "2024-08-14",
+          "hearingTime": "14:00"
+        }
+      }
+    ],
+    "closedDayCount": "",
+    "currentEvent": "",
+    "firstHearingDate": "14 Aug 2024",
+    "dataVersion": 0,
+    "cicCaseSelectedDocument": {}
+  },
+  "errors": [
+    "One party must be selected."
```

Comment:
Should an error be returned for this test? It seems like you are testing happy path here.

Reply:
This is just a generic response. This is not the output from the test case. I can remove the errors section if it is confusing.